app/src/main/java/jaelson/wagner/silva/upcomingmovies/data/model/movies/MovieDetailsResult.java

```diff
@@ -0,0 +1,67 @@
+package jaelson.wagner.silva.upcomingmovies.data.model.movies;
+
+import com.squareup.moshi.Json;
+import jaelson.wagner.silva.upcomingmovies.data.model.common.BaseResult;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jaelsonwagner@gmail.com.
+ */
+
+public class MovieDetailsResult extends BaseResult {
```

Comment:
Why map all the unused fields?

Reply:
I was thinking to use flatMap and call some function to extract errors. But, given my time, I have not validated my initial idea.